tests/test-recipes/metadata/noarch_python/noarch_python_test_package/README

```diff
@@ -0,0 +1 @@
+Simple package to test noarch package building.
```

Comment:
not really necessary, unless you're going to go into more detail about what this test package does exactly